### DISCUSSION TITLE
Firefox .pdf meeting list output error

### DISCUSSION THREAD

**Discussion by John-L** [code4recovery/12-step-meeting-list#1136] [Q&A]
The latest versions of Firefox (Windows & Android) may have a bug that causes .pdf printed output of meeting lists to display one column per page, rather than two.

Could someone consider verifying this, perform some error isolation, and work with developers at Firefox (if need be)?

The error does not occur using Edge, Chrome, or Opera.

Thanks in advance.

**Comment by kiyote33** [comment]
Hi @John-L,
I've tested this on Firefox version 112.0.2 and while the PDF Meeting List does default in a single column, it is a simple matter to click on the tools menu and change the Spreads option from No Spreads to Odd Spreads which will give you the desired two column display. As this is not a Code For Recovery issue but a feature of Firefox I will be closing this discussion.

**Reply by John-L** [reply]
Is there any place appropriate this gem could be noted for reference by the next person to stumble upon this (other than this forum)?

Thanks for your insight.

**Reply by John-L** [reply]
Where exactly is the tools menu in Firefox? I don't see it on the PC nor Android versions.

**Reply by kiyote33** [reply]
![image](https://github.com/code4recovery/12-step-meeting-list/assets/57125427/4e463294-15fb-4ae2-bb02-893d304bee74)
The red arrow points to it...

**Reply by John-L** [reply]
The site I use uses the MeetingFinder WordPress plug-in...no tools menu.

Is this control changeable by a site admin?

**Reply by kiyote33** [reply]
Hi John,
I'm assuming when you say "MeetingFinder WordPress plug-in" that you are referring to the "12 Step Meeting List" plug-in from Code For Recovery. Our plug-in does have settings which can be managed by a site admin. Your original question here concerned another of our products: the "PDF" Service which is designed to work with our 12 Step Meeting List plug-in to generate a PDF version of your Meeting List. Other than pulling the data from the WordPress tables for the 12 Step Meeting List, there is no connection between the two products. Hope this helps...

**Reply by John-L** [reply]
![Screenshot 2023-05-21 143017](https://github.com/code4recovery/12-step-meeting-list/assets/7512058/1cfa7447-587a-480b-9085-955b6a5910ce)
Here's a portion of the web-site I use. I simply use Firefox + Menu + Print... (or Cntl+P) to invoke the problem by spooling to a .pdf printer file.  The menu along the top is specific to our Area. Perhaps my user inexperience with your tool is showing...

**Reply by John-L** [reply]
If you could provide specific directions that I could forward to our site admin, I'd appreciate it.

**Comment by kiyote33** [comment]
Ha! This is a horse of a different color. You are simply using the built-in browser print mechanism to publish a text version of the Meeting List. Each browser may handle that differently. You might try playing around with the print setting through the browser to see if you can get an output suitable to your needs.
I would suggest you might ask your site admin to setup our PDF service. There's documentation here at https://pdf.code4recovery.org/ to help get it setup correctly. You can check [this link](https://edmontonalano.org/wp-content/uploads/documents/AlanoMeetingDirectory.pdf) for an example where I setup a printable PDF output for a club in my area:

**Reply by John-L** [reply]
Is the method of invoking your service the same? The output is formatted in such a manner that I suspect your print service is already installed, some how, some way. 

If so, please forward configuration info to produce the two column output from the print service.
